clang/lib/Sema/SemaDeclCXX.cpp

```diff
@@ -2393,6 +2393,25 @@ static bool CheckConstexprFunctionBody(Sema &SemaRef, const FunctionDecl *Dcl,
                                              Kind))
             return false;
       }
+    } else if (!Constructor->isDelegatingConstructor()) {
+      for (const Decl *decl : RD->decls()) {
+        if (const auto *inner = dyn_cast<CXXRecordDecl>(decl)) {
```

Comment:
Code-style NIT, the names of the variables should start from a capital. 
```suggestion
        if (const auto *Inner = dyn_cast<CXXRecordDecl>(Decl)) {
```